app/models/restaurant.rb

```diff
@@ -0,0 +1,7 @@
+class Restaurant < ApplicationRecord
+  validates :name, presence: true
+  validates :location, presence: true
+  validates :description, presence: true
+  validates :description, length: {minimum: 20}
```

Comment:
With each of these unaddressed Hound style violations, I grow increasingly reluctant to merge this PR...